### ISSUE TITLE
False positive finish-args-unnecessary-appid-own-name?

### ISSUE THREAD

**Issue by stbergmann** [flathub/flatpak-builder-lint#33]
A test build of PR [Update to LibreOffice 7.4.2](https://github.com/flathub/org.libreoffice.LibreOffice/pull/195) initiated earlier today failed at <https://buildbot.flathub.org/#/builders/19/builds/19> with
```
flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions org.libreoffice.LibreOffice.json
 in dir /srv/buildbot/worker/build-x86_64-1/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions org.libreoffice.LibreOffice.json'
 using PTY: False
{
    "errors": [
        "finish-args-unnecessary-appid-own-name"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
program finished with exit code 1
```
I assume the linter complains about the line <https://github.com/flathub/org.libreoffice.LibreOffice/blob/ebf2dbae06f39e50be7d62e11eed6e8724a109c5/org.libreoffice.LibreOffice.json#L724>
```
        "--own-name=org.libreoffice.LibreOfficeIpc0",
```
but it is unclear to me why it does that.  (And removing that line from the manifest would break the resulting LibreOffice, it would no longer start.)

**Comment by hfiguiere** [comment]
it's a bug, indeed.

**Comment by hfiguiere** [comment]
The check:
```python
                if own_name.startswith(appid):
```
Which matches but is wrong.

**Comment by hfiguiere** [comment]
> Have another example;
> 
> https://buildbot.flathub.org/#/builders/6/builds/248

it's not that linter rule.

**Comment by Eonfge** [comment]
My bad, I misread. I'll follow it up with another issue; https://github.com/flathub/flatpak-builder-lint/issues/37

**Comment by stbergmann** [comment]
Thanks for the fix.  Is there an ETA when it will be rolled out for github.com/flathub/... PRs?

**Comment by barthalion** [comment]
@stbergmann It has been just published.

**Comment by stbergmann** [comment]
...and seen solving my issue; thanks!
